wordpress-mobile/AztecEditor-iOS
# Implements cite element

Fixes #963 

This PR implements support for the <cite> element that is used on Gutenberg for quotes blocks.

To test:
 - Start the demo app
 - Check the demo content for Gutenberg that uses a cite.
